src/Controller/SingleRecipeController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class SingleRecipeController extends AbstractController
+{
+    /**
+     * @Route("/single/recipe", name="single_recipe")
+     * @param Request $request
+     * @return Response
+     */
+    public function index(Request $request)
+    {
+        $name = $request->query->get('name');
+        $imageUrl = $request->query->get('imageUrl');
+        $ingredients = $request->query->get('ingredients');
+        $instructions = $request->query->get('instructions');
+
+        return $this->render('single_recipe/index.html.twig', [
```

Comment:
nelabai suprantu sito actionio prasmes. Tu paduodi visus parametrus url'e vien tam kad atvaizduot template?

Reply:
ne geriau butu paduot `id` ir pagal tai pagetint is duombazes ir grazint recepto objekta i twig'a. o jei nerasta, mest NotFoundExceptiona?

Reply:
o jeigu `$recipe === null` twige failins. throwink not found exceptiona